Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -132,6 +132,52 @@ export async function findSigningIdentity(keychainPath: string) {
     }
 }
 
+/**
+ * Check if provisioning profile contains Cloud entitlement 
+ * @param provProfilePath
+ * @returns {boolean} 
+ */
+export async function cloudEntitlement(provProfilePath: string) {
+    //find the provisioning profile details
+    let provProfileDetails: string;
+    let getProvProfileDetailsCmd: ToolRunner = tl.tool(tl.which('security', true));
+    getProvProfileDetailsCmd.arg(['cms', '-D', '-i', provProfilePath]);
+    getProvProfileDetailsCmd.on('stdout', function (data) {
+        if (data) {
+            if (provProfileDetails) {
+                provProfileDetails = provProfileDetails.concat(data.toString().trim().replace(/[,\n\r\f\v]/gm, ''));
+            } else {
+                provProfileDetails = data.toString().trim().replace(/[,\n\r\f\v]/gm, '');
+            }
+        }
+    })
+    
+    await getProvProfileDetailsCmd.exec();
+
+    let tmpPlist: string;
+    if (provProfileDetails) {
+        //write the provisioning profile to a plist
+        tmpPlist = '_xcodetasktmp.plist';
+        fs.writeFileSync(tmpPlist, provProfileDetails);
+    } else {
+        throw tl.loc('ProvProfileDetailsNotFound', provProfilePath);
+    }
+
+    //use PlistBuddy to figure if cloud entitlement exists. 
+    let cloudEntitlement: string = await printFromPlist('Entitlements:com.apple.developer.icloud-container-environment', tmpPlist);
+
+    //delete the temporary plist file
+    let deletePlistCommand: ToolRunner = tl.tool(tl.which('rm', true));
```

Comment:
Above you use `fs` to create the file, why use a ToolRunner to delete it? Can you not also use `fs` for that?